packages/android-client/FishjamClient/src/main/java/com/fishjamcloud/client/FishjamClient.kt

```diff
@@ -117,7 +120,7 @@ class FishjamClient(
    * @param trackId an id of a valid local track that has been created using the current client
    * @return a boolean whether the track has been successfully removed or not
    */
-  fun removeTrack(trackId: String): Boolean = client.webrtcClient.removeTrack(trackId)
+  suspend fun removeTrack(trackId: String) = client.removeTrack(trackId)
```

Comment:
Because imo it's not useful to return a boolean here and I don't even know why it was here in the first place. If user gets returned false here, what are they supposed to do? Try again or what?